scripts/exregional_run_enkf.sh

```diff
@@ -376,73 +376,88 @@ use_gfs_nemsio=.true.,
 	lobsdiag_forenkf=.false.,
 	write_spread_diag=.false.,
 	netcdf_diag=${netcdf_diag:-.false.},
-        fv3_native=.true.,
+    fv3_native=.true.,
```

Comment:
Line 379 is not aligned.

Reply:
Thanks.  I changed from space to indent.